generator/src/main/java/com/graphicsfuzz/generator/tool/KnownValueShaderGenerator.java

```diff
@@ -124,7 +123,7 @@ public static void mainHelper(String[] args) throws ArgumentParserException, IOE
     final float bFloat = ns.getFloat("b");
     final float aFloat = ns.getFloat("a");
     final int maxUniforms = ns.getInt("max_uniforms");
-    final boolean generateUniformBindings = ns.getBoolean("generate_uniform_bindings");
+    final boolean generateUniformBindings = ns.getBoolean("vulkan");
```

Comment:
Rename `generateUniformBindings` to `isVulkan`.

Reply:
dang, I missed this one. done